src/cli/CommandsWinstonLogger.ts

```diff
@@ -0,0 +1,17 @@
+import winston, { Logger, transport } from 'winston'
```

Comment:
why do you call it "Winston" ? it is a logger object, exposing our debug/info/warn/error functions.
this one is ConsoleLogger

Winston is implementation issue - that is, not the business of anyone using it.
 (and actually, I don't see the benefit of using Winston in this case over "loglevel" - maybe the DEBUG/INFO prefixes..)

Reply:
Returned type is fixed to be `LoggerInterface`. This file creates a Winston logger though.